backend/src/api/api.ts

```diff
@@ -114,6 +116,13 @@ const initApi: FastifyPluginAsync = async (fastify) => {
     },
     prefix: ApiPath.MENTORS,
   });
+
+  fastify.register(initTasksApi, {
+    services: {
+      task,
+    },
+    prefix: ApiPath.TASKS,
+  });
```

Comment:
I think we need coach's advice, because I think that these api should be inside module api because it is a part of it's logical functionality (from my point of view)

Reply:
I think it's better to have a separate one, to leave a possibility to change it in future, if needed. Also, I'd prefer using different routes for different tables, so that everything is separated and grouped well
But let's see, what coaches think about that

Reply:
right, should be a separate `api`